packages/react/src/components/Storage/FileUploader/Previewer/__tests__/Previewer.test.tsx

```diff
@@ -4,9 +4,81 @@ import { render } from '@testing-library/react';
 import { Previewer } from '..';
 
 describe('Previewer', () => {
+  let fakeFile;
+  let fakeFile2;
+  beforeEach(() => {
+    fakeFile = new File(['hello'], 'hello.png', { type: 'image/png' });
+    fakeFile2 = new File(['goodbye'], 'goodbye.png', {
+      type: 'image/png',
+    });
+  });
   it('exists', async () => {
-    const { container } = render(<Previewer />);
+    const { container } = render(
+      <Previewer
+        acceptedFileTypes={['.png']}
+        files={[fakeFile]}
+        inDropZone={false}
+        multiple={true}
+        onClose={() => ''}
+        onDragEnter={() => ''}
+        onDragLeave={() => ''}
+        onDragOver={() => ''}
+        onDragStart={() => ''}
+        onDrop={() => ''}
+        onFileChange={() => ''}
+        fileNames={[]}
+        level="public"
+      />
+    );
 
     expect(container).toBeTruthy();
```

Comment:
As mentioned in previous PR, IMO truthy tests do not add much and is better replaced with snapshot tests. 

These can be very useful because it can catch visual regressions, which we saw happen with `PasswordField` in Vue/Angular for example.

Reply:
+1 to adding snaphot test. I would also add a couple assertions for child elements being rendered (query by classnames, `amplify-fileuploader__previewer` and `amplify-fileuploader__footer` etc)

Reply:
OK, I'll add it as a snapshot instead. I'll follow up in a different PR with more tests targeting the previewer and footer